client/dashboard/profile-wizard/index.js

```diff
@@ -48,10 +62,51 @@ export default class ProfileWizard extends Component {
 		return currentStep;
 	}
 
+	goToNextStep() {
+		const currentStep = this.getCurrentStep();
+		const currentStepIndex = getSteps().findIndex( s => s.key === currentStep.key );
+		const nextStep = getSteps()[ currentStepIndex + 1 ];
+
+		if ( 'undefined' === nextStep ) {
+			this.updateProfile( { complete: true } );
+		}
+
+		return updateQueryString( { step: nextStep.key } );
+	}
+
+	updateProfile( params ) {
+		const { addNotice } = this.props;
+		const payload = stringifyQuery( params );
+
+		return apiFetch( {
```

Comment:
Should we also convert this to a proper `wc-api` set of selectors/mutator actions? We'll want to be able to rely on flags like 'completed' and 'skipped' to hide the profiler from a GET call, so having state reflected by updates would be good. See https://github.com/woocommerce/woocommerce-admin/blob/master/client/dashboard/index.js#L25.

Reply:
Absolutely- good idea!  Created an issue around that [here](https://github.com/woocommerce/woocommerce-admin/issues/2303).